### PR TITLE
feat: Norwegian language

### PR DESCRIPTION
**Pull request by reytarovskiy**
### Please check if the PR fulfills these requirements
- [x] It's the right issue type on the title
- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [x] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes/features)
- [x] Docs have been added/updated (for bug fixes/features)
- [x] It does not introduce a breaking change or has a description of the breaking change

### Description
Adds the Norwegian language translations, updating README as well.

---
Thank you for your contribution to TOAST UI product. 🎉 😘 ✨

### CODE REVIEW

File: src/js/langs/nb_NO.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    'Remove col': 'Fjern kolonne',
+    'Align left': 'Venstreorienter',
+    'Align center': 'Senterorienter',
+    'Align right': 'Høyreorienter',
+    'Remove table': 'Fjern diagram',
+    'Would you like to paste as table?': 'Ønsker du å lime inn som et diagram?',
+    'Text color': 'Tekstfarge',
+    'Auto scroll enabled': 'Auto-scroll aktivert',
+    'Auto scroll disabled': 'Auto-scroll deaktivert',
+    'Choose language': 'Velg språl'
```
Comment (seonim-ryu) [inline src/js/langs/nb_NO.js]: Code convention is not right. Running the command below shows that the [indentation is incorrect](https://eslint.org/docs/rules/indent) in file `nb_NO.js`.
``` sh
npm run lint
```
error

Our code convention is indent `2`.

File: README.md [hunk 2]
```diff
@@ -97,7 +97,7 @@ Depending on the type of use you want like production of *Markdown* or maybe to
 
 ### And More
 
-* **i18n**: English, Dutch, Korean, Japanese, Chinese, Spanish, German, Russian, French, Ukrainian, Turkish, Finnish, Czech, Arabic, Polish, Galician, Swedish, Italian + language you extend.
+* **i18n**: English, Dutch, Korean, Japanese, Chinese, Spanish, German, Russian, French, Ukrainian, Turkish, Finnish, Czech, Arabic, Polish, Galician, Norwegian, Swedish, Italian + language you extend.
```
Comment (seonim-ryu) [inline README.md]: Can you move it after _Italian_? This is not the alphabetical order, but the order in which languages ​​are added.
```
... Swedish, Italian, Norwegian + ...
```

File: .github/ISSUE_TEMPLATE/stale.yml [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+staleLabel: inactive
+
+# Comment to post when marking as stale. Set to `false` to disable
+markComment: >
+  This issue has been automatically marked as inactive because there hasn’t been much going on it lately.
+  It is going to be closed after 7 days. Thanks!
+
+# Comment to post when closing a stale Issue or Pull Request.
+closeComment: >
+  This issue will be closed due to inactivity. Thanks for your contribution!
```
Comment (seonim-ryu) [inline .github/ISSUE_TEMPLATE/stale.yml]: This file has changed path [here](https://github.com/nhn/tui.editor/tree/master/.github). I think you fork before the file path changes. Please remove this file.

**Comment by reytarovskiy** [comment]
Thanks. I added your changes

**Comment by reytarovskiy** [comment]
When you plan to add this to the release?

**Comment by seonim-ryu** [comment]
@reytarovskiy 
We're going to take our next release on a long-term schedule, so there's no one yet.. I think it'll be early next year.

**Pull request merged**